docs/supported-metrics.md

```diff
@@ -54,3 +54,48 @@ In Mesmer we support 3 types of metrics:
 | Stream throughput            | counter |
 | Operator throughput          | counter |
 | Operator processing time     | counter |
+
+## ZIO (both Default-JVM-Metrics and others)
```

Comment:
General idea for this page - let's group the metrics in the following sections with different level headers:
```
# Supported metrics
## Akka
### (all current Akka sections)
## ZIO
### (all ZIO sections)
```
and in addition I'd split ZIO metrics into three sections like:
```
### Executor Metrics
### JVM Metrics
### ZIO Runtime Metrics

Reply:
@jczuchnowski, I refactored the fix in this Commit:
https://github.com/ScalaConsultants/mesmer/pull/470/commits/377f11eab460818b07249fe0b5c759f214d45413